utils/flare.py

```diff
@@ -123,6 +124,8 @@ def __init__(self, cmdline=False, case_id=None):
         self._hostname = get_hostname(self._config)
         self._prefix = "datadog-{0}".format(self._hostname)
 
+        self._proxy_settings = proxy.get_proxy(self._config)
```

Comment:
Actually, at this point, the `get_config()` call above has already populated `self._config` with the `proxy_settings`, so you can simply access them with `self._config.get('proxy_settings')`, no need to call `get_proxy` and store them separately here.